tests/CMakeLists.txt

```diff
@@ -374,19 +374,8 @@ add_test(NAME validate-secondary-interface
     --firmware ${PROJECT_SOURCE_DIR}/tests/test_data/firmware.txt)
 
 
+add_subdirectory(sota_tools)
 if(BUILD_SOTA_TOOLS)
-    # sota-tools tests
-    add_aktualizr_test(NAME sota_tools_auth_test SOURCES sota_tools/auth_test.cc
-                       PROJECT_WORKING_DIRECTORY NO_VALGRIND)
-    target_link_libraries(t_sota_tools_auth_test sota_tools_static_lib aktualizr_static_lib ${TEST_LIBS})
-    target_include_directories(t_sota_tools_auth_test
-                               PUBLIC ${PROJECT_SOURCE_DIR}/src/sota_tools ${GLIB2_INCLUDE_DIRS})
-
-    add_aktualizr_test(NAME ostree_hash SOURCES sota_tools/ostree_hash_test.cc)
-    target_link_libraries(t_ostree_hash sota_tools_static_lib)
-    target_include_directories(t_ostree_hash
-                               PUBLIC ${PROJECT_SOURCE_DIR}/src/sota_tools ${GLIB2_INCLUDE_DIRS})
-
     # Setup coverage for sota_tools_static_lib
```

Comment:
Any reason not to put the entire set of sota_tools tests in the tests/sota_tools CMakeLists? I'm wouldn't hold up the PR for that, but just wondering if there was a reason.

Reply:
None really.  I just moved the easy/obvious ones to start with.  We really need to sort through the stuff in tests/ though